### PR TITLE
James 3489 Explicit behavior of ValidRcptHandler in case of LDAP failure

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJamesServerTest.java [hunk 1]
```diff
@@ -80,4 +83,12 @@ void mailsShouldBeWellReceivedBeforeFirstUserConnectionWithLdap(GuiceJamesServer
             .select("INBOX")
             .awaitMessage(calmlyAwait);
     }
+
+    @Test
+    void receivingMailShouldIssueAnSmtpErrorWhenLdapIsNotAvailable(GuiceJamesServer server) {
+        ldap.getLdapRule().stop();
```
Comment (Arsnael) [inline server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJamesServerTest.java:89]: if this test gets played before the other ones defined by the contract, does ldap container stays in the stop state?

**Comment by Arsnael** [comment]
Thanks for this @rouazana :)

File: server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJamesServerTest.java (hunk 1 above)
Comment (rouazana) [reply server/container/guice/cassandra-ldap-guice/src/test/java/org/apache/james/CassandraLdapJamesServerTest.java]: fixed

**Review by LanKhuat — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**